package.json

```diff
@@ -98,7 +98,7 @@
     "@hot-loader/react-dom": "^16.13",
```

Comment:
Spacing
![image](https://user-images.githubusercontent.com/64950861/148084543-2ca9d1a1-f5e4-4398-a8e8-adebd06db582.png)

Reply:
I'm unable to reproduce this issue. This is the search result page, correct? I've tested this with multiple devices, browsers & resolutions. This is how it looks for me:
![image](https://user-images.githubusercontent.com/94839278/148097696-cb1874bd-2a3b-4b96-88d2-93d71e7739f4.png)

What's your setup?

Reply:
Is it possible that your browser is using cached CSS files for some reason?

Reply:
Sorry, should have mentioned it was in `https://jimbo.lbry.tv/$/following/discover` page.  Other places look fine as shown in your screen shot.

Reply:
Fixed.